### DISCUSSION TITLE
More detailed topology view?

### DISCUSSION THREAD

**Discussion by tobicodesstuff** [mattieserver/netbox-topology-views#298] [Ideas]
Hello,

firstly thank you for the plugin, it looks and works great. I have one question, namely I can't seem to make the plugin display info such as a cable name, device interface names and so on. It displays the solid line as expected but I have requests for a much more detailed view.

Can this be achieved and can I help with something? Sadly is JS not my field of expertise.

Best regards.

**Comment by dreng** [comment]
I think, this question should have been answered with the feature requests you opened, haven't it?

**Reply by tobicodesstuff** [reply]
Yes, I've forgotten to close the discussion. My bad.
